shuup/xtheme/templates/shuup/xtheme/admin/font_create.jinja

```diff
@@ -0,0 +1,5 @@
+{% extends "shuup/admin/base.jinja" %}
+{% from "shuup/admin/macros/general.jinja" import single_section_form with context %}
+{% block content %}
+    {{ single_section_form("admin_font_family_form", form) }}
```

Comment:
change `"admin_font_family_form"` to `"admin_font_form"` so the button Save knows the form to submit.
The save button has a form id target to submit. it should match with the `view.context_object_name` + `_form`